src/pages/settings/Troubleshoot/TroubleshootPage.tsx

```diff
@@ -119,7 +119,7 @@ function TroubleshootPage({shouldStoreLogs}: TroubleshootPageProps) {
                     >
                         <View style={[styles.flex1, styles.mt5]}>
                             <View>
-                                <ClientSideLoggingToolMenu />
+                                <ClientSideLoggingToolMenu isViaTestToolsModal={false} />
```

Comment:
Works and looks well, awesome job
We could simplify the code 

Instead of using `isViaTestToolsModal` prop, we should use `backTo` param ([more details](https://github.com/Expensify/App/blob/942cad969430f17c5477ae519bf6c3508cb761f8/src/ROUTES.ts#L12))
Then we won't have to use prop drilling to pass `isViaTestToolsModal` to `ConsolePage`, instead you get the route to navigate back to from `route.params.backTo`

Reply:
> Can we do that we the existing prop isViaTestToolsModal?

We shouldn't add a new prop is the functionality we need is already implemented, we can just use `backTo`


> Also, for the share logs page, we'll still need isViaTestToolsModal to create the fallback route, don't we?

We will achieve effectively the same result with `backTo`, take a look at how it's used in [NewContactMethodPage](https://github.com/Expensify/App/blob/8da0384bcead36db31cb39c4c345aace9ce3473f/src/pages/settings/Profile/Contacts/NewContactMethodPage.tsx#L50) for example